templates/nzbget/index.ts

```diff
@@ -5,6 +5,20 @@ import {
 
 export default createTemplate({
   name: "Nzbget",
+  meta: {
```

Comment:
The screenshot is not very useful. I did a google search and couldn't find something good. Please create a service and take a screenshot manually.